practice_21_03_2019/Program.cs

```diff
@@ -1,21 +1,49 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace practice_21_03_2019
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Task 1");
             int[] emptyArr = CreateEmptyIntArray();
+            Console.WriteLine($"Empty array length: {emptyArr.Length}");
+            Console.WriteLine();
+
+            Console.WriteLine("Task 2");
+            CreateMixedArray();
+            Console.WriteLine();
+
+            Console.Read();
         }
 
         public static int[] CreateEmptyIntArray()
         {
             return new int[0];
         }
+
+        public static void CreateMixedArray()
+        {
+            dynamic[] mixedArray= new dynamic[3];
```

Comment:
great solution but we know about dynamic type nothing. I expected there other solution.
Anyway your way is also workable.